integration

```diff
@@ -912,6 +912,7 @@
   "UI-Lovelace-Minimalist/UI",
   "user2684/imou_life",
   "v1ack/lelight",
+  "vakio-ru/vakio_base_smart",
```

Comment:
Are you still associated with this organization?

Reply:
Yes